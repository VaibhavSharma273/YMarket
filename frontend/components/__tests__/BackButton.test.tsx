import * as React from 'react';
import renderer from 'react-test-renderer';

import BackButton from '../BackButton';

it(`renders correctly`, () => {
    const tree = renderer.create(<BackButton />).toJSON();

    expect(tree).toMatchSnapshot();
});
