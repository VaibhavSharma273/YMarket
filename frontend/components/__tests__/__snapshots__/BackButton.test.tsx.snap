// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessible={true}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "left": 4,
      "opacity": 1,
      "position": "absolute",
      "top": 30,
    }
  }
>
  <Image
    source={1}
    style={
      Object {
        "height": 24,
        "width": 24,
      }
    }
  />
</View>
`;
